### ISSUE TITLE
Endpoint to allow loading of DCNA data across molecular profile

### ISSUE THREAD

**Issue by alisman** [cBioPortal/icebox#199]
Right now we can only load DCNA data for single molecular profile at a time.  On a multi-study query, this results in at least a single call per study.  Instead api should accept a list of profile/sample list id pairs

[32 study query](https://www.cbioportal.org/results/oncoprint?cancer_study_list=laml_tcga_pan_can_atlas_2018%2Cacc_tcga_pan_can_atlas_2018%2Cblca_tcga_pan_can_atlas_2018%2Clgg_tcga_pan_can_atlas_2018%2Cbrca_tcga_pan_can_atlas_2018%2Ccesc_tcga_pan_can_atlas_2018%2Cchol_tcga_pan_can_atlas_2018%2Ccoadread_tcga_pan_can_atlas_2018%2Cdlbc_tcga_pan_can_atlas_2018%2Cesca_tcga_pan_can_atlas_2018%2Cgbm_tcga_pan_can_atlas_2018%2Chnsc_tcga_pan_can_atlas_2018%2Ckich_tcga_pan_can_atlas_2018%2Ckirc_tcga_pan_can_atlas_2018%2Ckirp_tcga_pan_can_atlas_2018%2Clihc_tcga_pan_can_atlas_2018%2Cluad_tcga_pan_can_atlas_2018%2Clusc_tcga_pan_can_atlas_2018%2Cmeso_tcga_pan_can_atlas_2018%2Cov_tcga_pan_can_atlas_2018%2Cpaad_tcga_pan_can_atlas_2018%2Cpcpg_tcga_pan_can_atlas_2018%2Cprad_tcga_pan_can_atlas_2018%2Csarc_tcga_pan_can_atlas_2018%2Cstad_tcga_pan_can_atlas_2018%2Ctgct_tcga_pan_can_atlas_2018%2Cthym_tcga_pan_can_atlas_2018%2Cthca_tcga_pan_can_atlas_2018%2Cucs_tcga_pan_can_atlas_2018%2Cucec_tcga_pan_can_atlas_2018%2Cuvm_tcga_pan_can_atlas_2018&Z_SCORE_THRESHOLD=2.0&RPPA_SCORE_THRESHOLD=2.0&profileFilter=mutations%2Cfusion%2Cgistic&case_set_id=all&gene_list=CDKN2A%2520MDM2%2520MDM4%2520TP53&geneset_list=%20&tab_index=tab_visualize&Action=Submit)

You will see that ```https://www.cbioportal.org/api/molecular-profiles/acc_tcga_pan_can_atlas_2018_gistic/discrete-copy-number/fetch?discreteCopyNumberEventType=HOMDEL_AND_AMP&projection=DETAILED``` is called 32 times, once for each study.  This is probably not optimal.   But as far as I can tell, we don't have an endpoint that would allow us to request all of these at once

**Comment by alisman** [comment]
@jjgao @inodb maybe this is opportunity to try to set up a graphql solution to aggregate these requests/responses for the purpose of frontend.

**Comment by dippindots** [comment]
@alisman Could you elaborate on these: 

- which endpoint should we improve? I guess we want to make that endpoint to receive a list of profile/sample list id pairs instead of single profileId, right?
- could you provide a link for DCNA data

**Comment by jjgao** [comment]
Good idea to try to support multi-study. Not sure if we should go for graphql tho...
